urts/src/lib.rs

```diff
@@ -32,20 +67,24 @@ pub struct EnclaveBuilder {
 
     // `true` if the enclave should be created in debug mode
     debug: bool,
+
+    // Sealed key to use with Intel Protected Code Loader. None if PCL disabled.
+    pcl_key: Option<Vec<u8>>,
+
+    // Configuration to use with Key Separation & Sharing. None if KSS disabled.
+    kss_config: Option<KssConfig>,
 }
 
 impl EnclaveBuilder {
     /// Returns an EnclaveBuilder for the provided signed enclave.
     ///
     /// # Arguments
     ///
-    /// * `bytes` - The bytes representing the enclave file.  This should be a
-    ///   signed enclave.
-    pub fn new(bytes: &[u8]) -> EnclaveBuilder {
-        EnclaveBuilder {
-            bytes: bytes.into(),
-            debug: false,
-        }
+    /// * `path` - The path to the enclave file. This should be a signed
+    ///   enclave.
+    pub fn new(path: impl AsRef<Path>) -> std::io::Result<EnclaveBuilder> {
```

Comment:
Recommend just deleting this and leaving `TryFrom<T> where T: AsRef<Path>` be the way. Adding a usage example on `EnclaveBuilder` for this and the from bytes impls is going to be superior than having multiple paths to the same functionality anyways...

Reply:
Just to make sure I'm following: you want the only way to construct this type to be by using a `TryFrom` or a `From` from another type?

I can do that, I just don't love it from an expressiveness standpoint. I do get your point about having multiple paths to the same functionality, and I guess we do have this kind of `From`-first approach in a lot of other cases, but to me `EnclaveBuilder::new(thing)` is always going to be clearer in intent than `thing.from()`.

Reply:
Actually, you can't do a `TryFrom<T> where T: AsRef<Path>`, because of the blanket `TryFrom` impl in `core`. So I'm leaving `new` in place.